hasura/migrations/default/1674972789147_alter_table_public_circles_add_column_guild_role_id/down.sql

```diff
@@ -0,0 +1,4 @@
+-- Could not auto-generate a down migration.
+-- Please write an appropriate down migration for the SQL below:
+-- alter table "public"."circles" add column "guild_role_id" integer
+--  null;
```

Comment:
can we add down migrations. can we also squash both of these up migrations into one? @CryptoGraffe

Reply:
added